libs/ui/src/components/NeoLoading/NeoLoading.vue

```diff
@@ -1,8 +1,16 @@
+<template>
+  <OLoading v-bind="$attrs">
+    <slot>
+      <NeoIcon icon="spinner-third" spin size="large" class="has-text-white" />
+    </slot>
+  </OLoading>
+</template>
```

Comment:
this component is used in 2 loaders, will this affect the design?

Reply:
nope, it only replaces the default loader slot 

Loader.vue : normal 
![CleanShot 2023-11-16 at 13 33 37@2x](https://github.com/kodadot/nft-gallery/assets/44554284/882b0142-eebc-492a-afba-4133b68110b1)
loader.vue with not slot
![CleanShot 2023-11-16 at 13 34 10@2x](https://github.com/kodadot/nft-gallery/assets/44554284/ffe134d2-6303-4e2f-b408-d1ad93e28cff)
![CleanShot 2023-11-16 at 13 34 26@2x](https://github.com/kodadot/nft-gallery/assets/44554284/5afb87ba-9b53-49fd-b96d-33b72837851f)